source/fe/mapping_q.cc

```diff
@@ -1085,6 +1085,26 @@ void MappingQ<dim,spacedim>::transform
   MappingQ1<dim,spacedim>::transform(input, output, *q1_data, mapping_type);
 }
 
+template<int dim, int spacedim>
+void MappingQ<dim,spacedim>::transform
+(const VectorSlice<const std::vector< DerivativeForm<2, dim, spacedim> > > input,
+ VectorSlice<std::vector<Tensor<3,spacedim> > >             output,
+ const typename Mapping<dim,spacedim>::InternalDataBase &internal,
+ const MappingType type) const
+{
+  this->MappingQ1<dim,spacedim>::transform(input, output, internal, type);
```

Comment:
Just to be clear: that means that the current master is incorrect (my change shouldn't have changed the logic)?